src/concept-coach/launcher.cjsx

```diff
@@ -0,0 +1,171 @@
+# coffeelint: disable=max_line_length
+
+React = require 'react'
+BS = require 'react-bootstrap'
+_ = require 'underscore'
+classnames = require 'classnames'
+
+{channel} = require './model'
+
+
+Launcher = React.createClass
+  displayName: 'Launcher'
+  getInitialState: ->
+    isLaunching: false
+    isClosing: false
+
+  launch: ->
+    {isClosing} = @state
+    return if isClosing
+    @setState(isLaunching: true)
+    channel.emit('launcher.clicked')
+
+  close: ->
+    @setState(isClosing: true, isLaunching: false)
+
+  delayedClose: ->
+    _.delay( =>
+      @setState(isClosing: false)
+    , 800)
+
+  shouldComponentUpdate: (nextProps, nextState) ->
+    @state.isClosing isnt nextState.isClosing or @state.isLaunching isnt nextState.isLaunching
+
+  componentDidUpdate: ->
+    {isLaunching, isClosing} = @state
+    @delayedClose() if isClosing and not isLaunching
+
+  componentWillMount: ->
+    channel.on 'coach.unmount.success', @close
+
+  componentWillUnmount: ->
+    channel.off 'coach.unmount.success', @close
+
+  render: ->
+    {isLaunching, isClosing} = @state
+    height = '388px'
+    height = "#{window.innerHeight}px" if isLaunching and not isClosing
+
+    classes = classnames 'concept-coach-launcher',
+      launching: isLaunching
+      closing: isClosing
+
+    <div className='concept-coach-launcher-wrapper'>
+      <div className={classes} onClick={@launch}>
+        <svg x="0px" y="0px" width="100%" height={height}
+          preserveAspectRatio="xMidYMin meet" version="1.1" xmlns="http://www.w3.org/2000/svg">
+          <g>
+            <rect className='launcher-background' y="0.541" fill="#E5E5E5" width="100%" height="100%"/>
+            <rect className='launcher-desk' y="70%" fill-rule="evenodd" fill="#FFFFFF" width="100%" height="30%"/>
+          </g>
+        </svg>
+
+        <svg x="0px" y="0px" width="100%" height={height} viewBox="-100 0 1140 388"
```

Comment:
It looks like the only dynamic part of this SVG is the height. Could this SVG file be included in CSS rather than having it in the codebase? Or as a separate component that only renders the SVG?